lighthouse-core/scripts/chromium-web-tests/download-content-shell.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
+1 to calling this out in a `third-party/` folder if we don't intend to try to maintain it. I was halfway through reviewing the utils file until it occurred to me this was also copied verbatim.

If it's really dead in the source though and we're on our own moving forward, then I would propose simplifying a bit down to only what we really need.